### ISSUE TITLE
Trade up contract calculator (crafting)

### ISSUE THREAD

**Issue by bladegery** [bladegery/csgo-trader-extension#55]
Create something like:
http://csgo.exchange/contract/tradeup
It's a very low-priority  for me, so it's unlikely that I will be working on this anytime soon.
It's something that can be worked on independently be anyone though, feel free to take it.

**Comment by bladegery** [comment]
Added the non-coding tag because the hardest part of this is probably collecting the items, not implementing the formula.

**Comment by Okand1998** [comment]
Is there any chance you are still into this project? I have got the collecting the data part down but not the formula

**Comment by gergelyszabo94** [comment]
Not really, there are many excelent trade up calculators out there. For example: https://csfloat.com/trade-up If you end up making it thought, I can add it.
